br/pkg/restore/merge.go

```diff
@@ -106,22 +104,8 @@ func MergeFileRanges(
 		if leftKeys+rightKeys > splitKeyCount {
 			return false
 		}
-		// Do not merge ranges in different tables.
```

Comment:
Maybe we should avoid merge regions from different kv mode or keyspace in the future.

Reply:
OK, will handle this cases when tikv-br support different keyspaces and modes.